manifests/u/UltraDefragDevelopmentTeam/UltraDefrag/7.1.4/UltraDefragDevelopmentTeam.UltraDefrag.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: UltraDefragDevelopmentTeam.UltraDefrag
+PackageVersion: 7.1.4
+PackageLocale: en-US
+Publisher: UltraDefrag Development Team
+PackageName: UltraDefrag
+License: Copyright © 2007-2019 UltraDefrag Development Team
+ShortDescription: A disk defragmenter program.

```

Comment:
```suggestion
ShortDescription: UltraDefrag is a powerful disk defragmenter for Windows. It can quickly boost performance of your computer and is easy to use. Also it can automatically defragment your disks so you won't need to take care about that yourself.
```

Reply:
Not sure where you found that description, I try to search the websites for descriptions like that, but that is much better.